modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -14,6 +14,8 @@ import java.util.Date;
 
 import com.liferay.portal.kernel.service.ServiceContextThreadLocal;
 
+import com.liferay.portal.kernel.cookies.CookiesManagerUtil;
+
```

Comment:
@MicaelleSilvaa, please sort the imports in alphabetical order :wink:

Reply:
The import is sorted by the json object